node/src/proxy_server/mod.rs

```diff
@@ -576,6 +579,7 @@ impl ProxyServer {
                 "Reporting shutdown of {} to counterpart", &stream_key
             );
             let ibcd = InboundClientData {
+                timestamp: SystemTime::UNIX_EPOCH, // TODO: Drive this in
```

Comment:
This also seems to need your attention. 

If reflected in tests already, just remove the comment please.